ui/raidboss/data/04-sb/alliance/ridorana_lighthouse.js

```diff
@@ -846,14 +846,14 @@
         'Construct 7': 'Bâtisseur n°7',
         'Construct 7.1': 'Bâtisseur n°7 amélioré',
         'Dark Rain': 'Trombe d\'eau',
-        'Echoes from Time\'s Garden will be sealed off': 'Fermeture du Jardin d\'un autre temps dans',
+        'Echoes from Time\'s Garden': 'Jardin d\'un autre temps dans',
```

Comment:
No need to have _"dans"_ in this sentence I guess. The full sentence is _"Fermeture du Jardin d\'un autre temps dans 15 secondes."_ so _dans_ means _in_.

Reply:
Ah whoops! Thank you for catching my mistake.